cdap-docs/developers-manual/source/building-blocks/spark-programs.rst

```diff
@@ -249,6 +249,24 @@ Spark in Workflows
 Spark programs in CDAP can also be added to a :ref:`workflow <workflows>`, similar to a :ref:`MapReduce <mapreduce>`.
 
 
+Spark SQL
+=========
+
+As you might know the `entry point into all functionality in Spark SQL
```

Comment:
'As you might know' may not be the best way to start this section. @awholegunch, any better suggestions?